README.md

```diff
@@ -0,0 +1,38 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Abdulrahman's website
+
+Simple website with text
+
+![screenshot](./screenshot.png)
+
+Simple website
+
+## Learning Objectives
+
+- Create branches with different base branches.
+- Understand the concept of version control.
+- Follow Gitflow.
+- Open pull requests in GitHub.
+- Write descriptive, easy to understand commit messages.
+- Write short, detailed, easy to understand descriptions of pull requests.
+- Write a descriptive and easy to understand README in business English for every project submitted.
+- Demonstrate an ability to submit a project for code review.
+
```

Comment:
- [ ] **Required** Include other sections of the README such as:

- USAGE
- GETTING STARTED
- PREREQUISITE, etc.